bin/group.weights

```diff
@@ -0,0 +1,9 @@
+nnlib network weights data file version 1.0
```

Comment:
@Ky5595 Too much clutter in the bin folder. Could you group some of your files and use a sub-folder?